static/app/views/dashboards/manage/index.tsx

```diff
@@ -230,13 +245,29 @@ function ManageDashboards() {
   function renderActions() {
     const activeSort = getActiveSort();
     return (
-      <StyledActions>
+      <StyledActions listView={organization.features.includes('dashboards-table-view')}>
         <SearchBar
           defaultQuery=""
           query={getQuery()}
           placeholder={t('Search Dashboards')}
           onSearch={query => handleSearch(query)}
         />
+        <Feature features={'organizations:dashboards-table-view'}>
+          <SegmentedControl<DashboardsLayout>
+            onChange={setDashboardsLayout}
+            size="md"
+            value={dashboardsLayout}
+            aria-label="Layout Control"
```

Comment:
nit, but I think we should wrap aria-labels with `t()` because they're user-facing and should also match the user's language

Reply:
oooo yes good catch